From 0fd0f18347d5b37546a6d016377c5f68e43b5af1 Mon Sep 17 00:00:00 2001
From: Akira Ajisaka <aajisaka@apache.org>
Date: Wed, 14 Jan 2015 17:38:35 +0900
Subject: [PATCH 1350/2748] MAPREDUCE-6210. Use getApplicationAttemptId()
 instead of getApplicationID() for logging
 AttemptId in RMContainerAllocator.java
 (Contributed by Leitao Guo)

(cherry picked from commit d336d136785ef1e49e0a110a1b9f0d1824829877)
(cherry picked from commit 2e5eef9da23b7fe58876987c3348a4fba653f551)

Change-Id: Ia4f081d88b652bb37c513ee0d5956b9aacf03d18
---
 .../mapreduce/v2/app/rm/RMContainerAllocator.java  |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/rm/RMContainerAllocator.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/rm/RMContainerAllocator.java
index a41b9f2..64238d0 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/rm/RMContainerAllocator.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/rm/RMContainerAllocator.java
@@ -705,7 +705,7 @@ public void rampDownReduces(int rampDown) {
         JobEventType.JOB_AM_REBOOT));
       throw new RMContainerAllocationException(
         "Resource Manager doesn't recognize AttemptId: "
-            + this.getContext().getApplicationID(), e);
+            + this.getContext().getApplicationAttemptId(), e);
     } catch (ApplicationMasterNotRegisteredException e) {
       LOG.info("ApplicationMaster is out of sync with ResourceManager,"
           + " hence resync and send outstanding requests.");
-- 
1.7.9.5

